script/setup/install-runc

```diff
@@ -37,7 +37,7 @@ function install_runc() {
 	git clone "${RUNC_REPO}" "${TMPROOT}"/runc
 	pushd "${TMPROOT}"/runc
 	git checkout "${RUNC_VERSION}"
-	make BUILDTAGS='seccomp' runc
+	make BUILDTAGS='seccomp' VERSION="${RUNC_VERSION}" runc
```

Comment:
I think unsetting the VERSION in the context of install-runc will be better, so that it falls back to the behaviour of getting version as specified in the build script of runc.